src/NServiceBus.Unity/packages.config

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="CommonServiceLocator" version="1.3" targetFramework="net45" />
-  <package id="Fody" version="1.29.4" targetFramework="net452" developmentDependency="true" />
+  <package id="Fody" version="4.0.2" targetFramework="net452" developmentDependency="true" />
```

Comment:
Since I already mentioned the schema tweak, now I'm curious, why did you update Fody at all? You shouldn't have needed to.

Reply:
@bording it wasn't building without an update.

Reply:
Just build it a couple times, it will start working.

Reply:
you're right. I've reverted the fody package back to the old one.